thermostat_house:
  ros__parameters:
    thermostat_name : "thermostat"
    target_temperature: 30
    mode: "away"
